lib/hocon/impl/simple_config_list.rb

```diff
@@ -149,11 +216,122 @@ def render_value_to_sb(sb, indent_size, at_root, options)
     end
   end
 
-  def new_copy(origin)
-    Hocon::Impl::SimpleConfigList.new(origin, @value)
+  def contains?(o)
```

Comment:
I figured it wouldn't hurt anything, but I can delete them if we feel like they hurt more than help.